src/pydvl/value/shapley/montecarlo.py

```diff
@@ -370,14 +366,14 @@ def combinatorial_montecarlo_shapley(
 
```

Comment:
The two lines above (github wouldn't let me comment on those) are the kind of boilerplate that I think we should avoid, that I refer to in my previous comment. Is there a problem with `put()`ing all `map_kwargs` (which are not ints, bools, etc.)?

Reply:
Actually no. using `put()` on all `map_kwargs` and `reduce_kwargs` should not cause an issue. I will try that now.

Reply:
Done.